blueos_repository/consolidate.py

```diff
@@ -56,8 +73,16 @@ class Version:
     authors: List[Author]
     docs: Optional[str]
     readme: Optional[str]
-    company: Optional[Company]
+    maintainer: Optional[Maintainer]
```

Comment:
That's a big problem, we should avoid breaking the json description.

Reply:
I agree we should try to avoid backwards incompatible changes, but in this case `maintainer` is what that field was supposed to represent. We can make leave the `company` field in but force it to be empty/none if that’s better, but that just seems unnecessary.

I would note that extensions infrastructure is still in beta, so I think better to make these changes now than later.

Reply:
I think that it's worth to just move that to another PR if we are going to break the api, we should break it only once, and think more about the fields.

We should focus here about the tags.

Reply:
What happens if you have multipler authors for the same extension ?

Reply:
> I think that it's worth to just move that to another PR if we are going to break the api, we should break it only once, and think more about the fields.

Ok, that'll be some effort to pull out, but I'll try to do so tomorrow.

> What happens if you have multipler authors for the same extension ?

Not sure how that's relevant. I'm considering the `maintainer` as the repo owner, which could be an individual or an organisation. There's a separate tag for `authors` that's already a list, so there can be an arbitrary number of those, they just aren't all in charge of what gets merged in/released.

Reply:
The `maintainer` changes have been pulled out, and can be handled later.